docs/develop/01_sdk/01_quickstart.md

```diff
@@ -68,11 +68,10 @@ To enable ES modules in your project, add `"type": "module"` to the `package.jso
 
 Declare an `async main` function in the `quickstart.ts` file that executes the rest of the code in this quickstart and call the `main()` function by default:
 
-{/* TODO: Do we need to test this or provide JS/TS equivalent? */}
+{/_ TODO: Do we need to test this or provide JS/TS equivalent? _/}
```

Comment:
```suggestion
```

I think you have a linter messing this up, but we can probably just cut it anyway

Reply:
Yeah I thought it was because of prettier but linter is better candidate. But its fix int this:
https://github.com/KILTprotocol/docs/pull/329/commits/2fb4bf229be1bd8c28ed45066790a66d8b5f1e29